CHANGELOG.md

```diff
@@ -38,11 +38,11 @@ This new architecture is intended to make it easier to consume and work with Rea
 * None.
 
 ### Compatibility
-* This release is compatible with Kotlin 1.5.20 and Coroutines 1.5.0.
+* This release is compatible with Kotlin 1.5.10 and Coroutines 1.5.0.
 
 ### Internal
 * Updated `com.gradle.plugin-publish` to 0.15.0.
-* Updated Kotlin support to 1.5.20.
+* Reverted to Kotlin `1.5.10` to fix a compatibility issue.
```

Comment:
Since the previous versions didn't use 1.5.20 I guess we didn't revert. Only reverted for the snapshots.

Reply:
I'll remove the entry